src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -23,9 +23,6 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.SortDirection;
-//import java.util.ArrayList;
-//import java.util.List;
-//import java.util.UUID;
 import java.util.*;
 
```

Comment:
For some reason GitHub won't let me comment on the lines below, but! Can you add a javadoc comment to the getUsers function explaining what it returns? It's probably also worth mentioning that it iterates across all of the messages so that someone calling it is aware that it could be potentially expensive to call it repeatedly.

Reply:
resolved. See my new commit to this branch